elements/rh-tile/rh-tile.ts

```diff
@@ -0,0 +1,236 @@
+ * @cssprop --rh-tile-text-color - color of text - {@default var(--rh-color-text-primary-on-light, #151515)}
+ * @cssprop --rh-tile-text-color-secondary - disabled text and icons - {@default var(--rh-color-text-secondary-on-light, #4d4d4d)}
+ * @cssprop --rh-tile-interactive-color - color of interactive elements - {@default var(--rh-color-border-interactive-on-light, #0066cc)}
+ * @cssprop --rh-tile-link-color - color of tile link - {@default var(--rh-tile-interactive-color)}
+ * @cssprop --rh-tile-link-text-decoration - tile link text decoration - {@default none}
+ * @cssprop --rh-tile-background-color - color tile surface - {@default var(--rh-color-surface-lightest, #ffffff)}
+ * @cssprop --rh-tile-focus-background-color - color tile surface on focus/hover - {@default var(--rh-color-surface-lighter, #f2f2f2)}
+ * @cssprop --rh-tile-disabled-background-color - color tile surface when disabled - {@default var(--rh-color-surface-light, #e0e0e0)}
+ * @cssprop --rh-tile-border-color - color of tile border - {@default var(--rh-color-border-subtle-on-light, #c7c7c7)}
+ */
+@customElement('rh-tile')
+export class RhTile extends LitElement {
+  static readonly styles = [styles];
+
+  private static readonly _disabledIcon = html`<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48"><g id="uuid-0fd9e805-a455-40ef-9171-f2f334832bf2"><rect width="48" height="48" fill="none"/></g><g id="uuid-48f9e284-0601-4fcd-bbe7-8b444234ac6c"><path d="m24,7c-9.37,0-17,7.63-17,17s7.63,17,17,17,17-7.63,17-17S33.37,7,24,7Zm15,17c0,3.52-1.23,6.76-3.27,9.32L14.68,12.27c2.56-2.04,5.8-3.27,9.32-3.27,8.27,0,15,6.73,15,15Zm-30,0c0-4.03,1.61-7.69,4.2-10.38l21.18,21.18c-2.7,2.6-6.35,4.2-10.38,4.2-8.27,0-15-6.73-15-15Z"/></g></svg>`;
+
+  /**
+   * whether tile interaction is disabled but retains ability to be read by screen readers;
+   * preferred method of disabling instead of `disabled`
+   */
+  @property({ reflect: true, attribute: 'aria-disabled', type: String }) ariaDisabled = 'false';
+
+  /**
+   * whether image is full-width (i.e. bleeds into the padding)
+   */
+  @property({ attribute: 'bleed', type: Boolean }) bleed = false;
+
+  /**
+   * whether headline link text is a desaturated color instead of blue;
+   * `true` sets headline color to white on dark tiles or black on light tiles
+   */
+  @property({ attribute: 'desaturated', type: Boolean }) desaturated = false;
+
+  /**
+   * reduces tile padding for more compact spaces
+   */
+  @property({ attribute: 'compact', type: Boolean }) compact = false;
+
+  /**
+   * namespace of icon
+   */
+  @property({ attribute: 'icon', type: String }) icon = false;
+
+  /**
+   * whether tile can be checked like a radio or checkbox:
+   * `false` (default) - tile behaves like a link;
+   * `true` - tile behaves like a checkbox unless it is part of an
+   * `rh-tile-group` with a `radio` type and more than one tile
+   */
+  @property({ attribute: 'checkable', type: Boolean }) checkable = false;
```

Comment:
```suggestion
  @property({
    converter: booleanEnumConverter({ values: ['radio'] })
  }) checkable = false;
```
consider making this boolean-or-value. 

tests:

```html
<rh-tile id=none></rh-tile>
<rh-tile id=bare interactive></rh-tile>
<rh-tile id=empty interactive=""></rh-tile>
<rh-tile id=silly interactive="silly"></rh-tile>
<rh-tile id=radio interactive="radio"></rh-tile>
<rh-tile id=case interactive="RaDio"></rh-tile>
<script>
  console.assert(none.interactive  === false); 
  console.assert(bare.interactive  === true); 
  console.assert(empty.interactive === true); 
  console.assert(silly.interactive === true); 
  console.assert(radio.interactive === 'radio');
  console.assert(case.interactive  === 'radio');
</script> 
```
if we don't already have a converter-constructor function for this in pfe-core or in `/lib`, now's the time, there are at least 2 other instances (perhaps in open PRs or in pfe, so please check with the team)

Reply:
It's deliberately not a radio unless it is in a group that sets it as such because we don't want single radio tiles.